engine_test.go

```diff
@@ -770,157 +764,157 @@ func TestStarPanic197(t *testing.T) {
-		sql.NewEmptyContext(),
-		"CREATE INDEX myidx ON mytable (i) WITH (async = false)",
-	)
-	require.NoError(t, err)
-
-	_, _, err = e.Query(
-		sql.NewEmptyContext(),
-		"CREATE INDEX myidx_multi ON mytable (i, s) WITH (async = false)",
-	)
-	require.NoError(t, err)
-
-	defer func() {
-		done, err := e.Catalog.DeleteIndex("mydb", "myidx", true)
-		require.NoError(t, err)
-		<-done
-
-		done, err = e.Catalog.DeleteIndex("foo", "myidx_multi", true)
-		require.NoError(t, err)
-		<-done
-	}()
-
-	testCases := []struct {
-		query    string
-		expected []sql.Row
-	}{
-		{
-			"SELECT * FROM mytable WHERE i = 2",
-			[]sql.Row{
-				{int64(2), "second row"},
-			},
-		},
-		{
-			"SELECT * FROM mytable WHERE i > 1",
-			[]sql.Row{
-				{int64(3), "third row"},
-				{int64(2), "second row"},
-			},
-		},
-		{
-			"SELECT * FROM mytable WHERE i < 3",
-			[]sql.Row{
-				{int64(1), "first row"},
-				{int64(2), "second row"},
-			},
-		},
-		{
-			"SELECT * FROM mytable WHERE i <= 2",
-			[]sql.Row{
-				{int64(2), "second row"},
-				{int64(1), "first row"},
-			},
-		},
-		{
-			"SELECT * FROM mytable WHERE i >= 2",
-			[]sql.Row{
-				{int64(2), "second row"},
-				{int64(3), "third row"},
-			},
-		},
-		{
-			"SELECT * FROM mytable WHERE i = 2 AND s = 'second row'",
-			[]sql.Row{
-				{int64(2), "second row"},
-			},
-		},
-		{
-			"SELECT * FROM mytable WHERE i = 2 AND s = 'third row'",
-			([]sql.Row)(nil),
-		},
-		{
-			"SELECT * FROM mytable WHERE i BETWEEN 1 AND 2",
-			[]sql.Row{
-				{int64(1), "first row"},
-				{int64(2), "second row"},
-			},
-		},
-		{
-			"SELECT * FROM mytable WHERE i = 1 OR i = 2",
-			[]sql.Row{
-				{int64(1), "first row"},
-				{int64(2), "second row"},
-			},
-		},
-		{
-			"SELECT * FROM mytable WHERE i = 1 AND i = 2",
-			([]sql.Row)(nil),
-		},
-	}
-
-	for _, tt := range testCases {
-		t.Run(tt.query, func(t *testing.T) {
-			require := require.New(t)
-
-			tracer := new(test.MemTracer)
-			ctx := sql.NewContext(context.TODO(), sql.WithTracer(tracer))
-
-			_, it, err := e.Query(ctx, tt.query)
-			require.NoError(err)
-
-			rows, err := sql.RowIterToRows(it)
-			require.NoError(err)
-
-			require.Equal(tt.expected, rows)
-			require.Equal("plan.IndexableTable", tracer.Spans[len(tracer.Spans)-1])
-		})
-	}
-}
-
-func TestCreateIndex(t *testing.T) {
-	require := require.New(t)
-	e := newEngine(t)
-
-	tmpDir, err := ioutil.TempDir(os.TempDir(), "pilosa-test")
-	require.NoError(err)
-
-	require.NoError(os.MkdirAll(tmpDir, 0644))
-	e.Catalog.RegisterIndexDriver(pilosa.NewIndexDriver(tmpDir))
-
-	_, iter, err := e.Query(sql.NewEmptyContext(), "CREATE INDEX myidx ON mytable (i)")
-	require.NoError(err)
-	rows, err := sql.RowIterToRows(iter)
-	require.NoError(err)
-	require.Len(rows, 0)
-
-	defer func() {
-		time.Sleep(1 * time.Second)
-		done, err := e.Catalog.DeleteIndex("foo", "myidx", true)
-		require.NoError(err)
-		<-done
-
-		require.NoError(os.RemoveAll(tmpDir))
-	}()
-}
+// func TestIndexes(t *testing.T) {
```

Comment:
shall we uncomment this code, even if it is failing? just to do not forget about that.

Reply:
Don't worry, I am fixing those tests in my PR. If he uncomments that code, the root package will not compile